Packages/Netherlands3D/WebServiceReader/Runtime/Scripts/Layer/WMSImageLayer.cs

```diff
@@ -1,23 +1,59 @@
+/*
+*  Copyright (C) X Gemeente
+*                X Amsterdam
+*                X Economic Services Departments
+*
+*  Licensed under the EUPL, Version 1.2 or later (the "License");
+*  You may not use this work except in compliance with the License.
+*  You may obtain a copy of the License at:
+*
+*    https://github.com/Amsterdam/Netherlands3D/blob/main/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" basis,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+*  implied. See the License for the specific language governing
+*  permissions and limitations under the License.
+*/
+
 using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using Netherlands3D.TileSystem;
 using System;
 using Netherlands3D.Core;
 using UnityEngine.Networking;
+using UnityEngine.Rendering.Universal;
 
 namespace Netherlands3D.Geoservice
 {
     public class WMSImageLayer : Layer
-    {
-        
-        private int BuildingStencilID=0;
-        private int TerrainStencilID = 0;
-        [SerializeField]
-        private int activeStencilID = 0;
-        private int activeStencilMask = 255;
+    {    
         public bool compressLoadedTextures = false;
-        public GameObject TilePrefab;
+        private bool prefabIsDecalProjector = false;
+        private GameObject tilePrefab;
+        public GameObject TilePrefab {
+            get
+            {
+                return tilePrefab;
+            }
+            set
+            {
+                tilePrefab = value;
+                var decalProjector = tilePrefab.GetComponent<DecalProjector>();
+                if(decalProjector != null)
+                {
+                    projectorMaterialAsset = decalProjector.material;
+                    prefabIsDecalProjector = true;
```

Comment:
Kan versimpeld naar `prefabIsDecalProjector = decalProjector != null`; scheelt mogelijk een else als we ook de material een ternary statement van maken